ES/ESmy016.xml

```diff
@@ -365,111 +166,110 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                               <dim type="left">12</dim>
                               <dim type="intercolumn"/>
                            </dimensions>
-                           <note>All data are for <locus target="#1r">1r</locus>.</note>
+                           <note>All data are for <locus target="#1r"/>.</note>
                            <ab type="ruling"/>
-                           <ab type="pricking">Pricking and ruling are partly visible. </ab>
-                           <ab type="ruling" subtype="pattern"> Ruling pattern: 1A-1A/0-0/0-0/C </ab>
-                           <ab type="pricking"> Primary pricks are partly visible. </ab>
-                           <ab type="pricking"> Ruling pricks are visible. </ab>
-                           <ab type="pricking"> Structural pricks are partly visible (<locus from="20v" to="21r">20v-21r</locus>).</ab>
-                           <ab type="ruling"> The upper line is written above the ruling.</ab>
-                           <ab type="ruling"> The bottom line is written above the ruling.</ab>
+                           <ab type="pricking">Pricking and ruling are partly visible.</ab>
+                           <ab type="ruling" subtype="pattern">1A-1A/0-0/0-0/C</ab>
+                           <ab type="pricking">Primary pricks are partly visible.</ab>
+                           <ab type="pricking">Ruling pricks are visible.</ab>
+                           <ab type="pricking">Structural pricks are partly visible (<locus from="20v" to="21r">20v-21r</locus>).</ab>
+                           <ab type="ruling">The upper line is written above the ruling.</ab>
+                           <ab type="ruling">The bottom line is written above the ruling.</ab>
                         </layout>
                         <layout>
-                           <ab type="punctuation" subtype="Executed">not quite regularly;</ab>
+                           <ab type="punctuation" subtype="Executed">not quite regularly.</ab>
                            <ab type="punctuation" subtype="Usage">not always logical.</ab>
                            <ab type="punctuation">
```

Comment:
Dividers is a subtitle (subtype), not part of list (see eg ESam007):

```
<ab type="punctuation" subtype="Dividers">
                              <list>
                                 
                                 <item xml:id="div2"> I. Colon;</item>
```

Reply:
I took it over from the initial xml version as it was and did not do anything with this part. I do not think that these details are needed in the catalogue.

Reply:
in the printed catalogue